flintrock/flintrock.py

```diff
@@ -156,20 +160,35 @@ def get_config_file() -> str:
     return config_file
 
 
+def configure_log(debug: bool):
+    handler = logging.StreamHandler()
+    handler.setLevel(logging.DEBUG)
+    if debug:
+        logger.setLevel(logging.DEBUG)
+        handler.setFormatter(logging.Formatter('%(asctime)s %(message)s'))
+    else:
+        logger.setLevel(logging.INFO)
+        handler.setFormatter(logging.Formatter('%(message)s'))
+    logger.addHandler(handler)
+
+
 @click.group()
 @click.option(
     '--config',
     help="Path to a Flintrock configuration file.",
     default=get_config_file())
 @click.option('--provider', default='ec2', type=click.Choice(['ec2']))
 @click.version_option(version=__version__)
+# TODO: implement some solution like in https://github.com/pallets/click/issues/108
+@click.option('--debug/--no-debug', default=False, help="Whether to show debug messages")
```

Comment:
For consistency with the other help messages, I prefer: "Show debug information."